app/domain/errors.rb

```diff
@@ -353,4 +353,18 @@ module Util
       code: "CONJ00044E"
     )
   end
+
+  module System
+    EmptyConjurDataKey = ::Util::TrackableErrorClass.new(
+      msg:  "CONJUR_DATA_KEY is empty. " \
+            "Try loading the data key as an environment variable.",
+      code: "CONJ00065E"
```

Comment:
is this the correct code? not saying it isn't, just want to verify it doesn't exist already